src/views/registration-js/attendees/fields/email.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * This template renders a Single Ticket content
+ * composed by Title and Description currently
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/registration-js/attendees/fields/email.php
+ *
+ * @since  TBD
+ * @version TBD
+ *
+ * @see Tribe__Tickets_Plus__Meta__Field__Email
+ *
+ */
+
```

Comment:
:no_entry_sign: Additional blank lines found at end of doc comment

<sub>hash: 9a40d91f9814f16c41fab06af1c13c7f</sub>